src/fcontext/fcontext.c

```diff
@@ -4,6 +4,21 @@
 #include <stdlib.h>
 #include <assert.h>
 
+// Lifted from https://github.com/llvm/llvm-project/blob/release/16.x/clang/lib/Headers/cet.h
+# ifdef __LP64__
+#  if __CET__ & 0x1
```

Comment:
Need to test for `#ifdef __CET__` before trying to use it in an expression.

Reply:
yes, my bad.